tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+skipsdist = true
+envlist = tests, flake8
+
+[testenv]
+passenv = PYTEST_ADDOPTS="--variables=/path/to/variables.json" tox
+deps =
+    axl==0.5.1
+    fxapom==1.9.0
+    pyjwt==1.4.2
+    pytest~=3.0.0
+    pytest-selenium
+    pytest-variables
+    pytest-xdist==1.15.0
+    pytz==2016.6.1
+    requests==2.11.1
+commands = pytest {posargs}
```

Comment:
I'm hitting an issue where the latest scipy (a dependency) binary is not recognised as compatible with my python version (2.7.12), which is causing it to compile from source. This takes a long time. I was able to resolve this by adding the following line to `tox.ini`:

```ini
install_command = pip install --only-binary scipy {opts} {packages}
```

I'm not suggesting we add this, but it could be useful if others face this problem.